03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,36 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <header>
+        <h1>Daily Planet </h1>
+        <nav>
+            <a href="www.sports.com">sports</a>
+            <a href="local.com">local</a>
+            <a href="www.economy">economy</a>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <article>
+                <h2> I Love Lamp </h2>
+                <p> Student loves lamp </p>
+                <img src="http://s7d1.scene7.com/is/image/BedBathandBeyond/3614813588384p" width="1" height="250" </>
```

Comment:
great nesting on this, but I would give a little more vertical space between elements and comment a bit more :)  also, try not to set the width and height of the image in the tag; best to do this with CSS. so far so good :)